R/methods.R

```diff
@@ -782,16 +782,11 @@ plot.vsel <- function(
   } else {
     baseline_pretty <- "best submodel"
   }
```

Comment:
`baseline_pretty` is not used anymore, it seems. So could be removed, I guess.

Reply:
removed